server/models/Deployment.ts

```diff
@@ -1,11 +1,14 @@
-import { Document, model, Schema, Types } from 'mongoose'
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Schema, model, Document, Types } from 'mongoose'
+import MongooseDelete from 'mongoose-delete'
 import logger from '../utils/logger'
 import { ModelDoc } from './Model'
 import { UserDoc } from './User'
 import { VersionDoc } from './Version'
 
 export const approvalStateOptions = ['Accepted', 'Declined', 'No Response']
 
+// eslint-disable-next-line no-shadow
```

Comment:
This is now being handled by the eslint config and can be removed

Reply:
Removed